pipelines/test/tw/emptypackage.yaml

```diff
@@ -0,0 +1,18 @@
+name: emptypackage
+
+pipeline:
+  - name: empty package check
+    runs: |
+      # Get our empty package name
+      pkg=$(basename ${{targets.contextdir}})
+
+      file_list=$(apk info --quiet --contents "$pkg" | grep -v "^$")
+
+      # We're only really expecting a single spdx.json file in an empty package
+      if [ $(echo "$file_list" | wc -l) -le 1 ] && (echo "$file_list" | grep -q ".spdx.json$"); then
+        echo "PASS: Package [$pkg] seems to be empty, as expected"
+      else
+        echo "FAIL: Expected this package [$pkg] to be empty, but it isn't:" >&
+        echo "$file_list" >&2
+        exit 1
+      fi
```

Comment:
I know it might seem like overkill, but can you follow the pattern of other tests, that have pipeline that just calls a program delivered in a package? 

ie, take a look at https://github.com/chainguard-dev/tw/pull/130 .  If you want to put your [metapackage](https://github.com/chainguard-dev/tw/pull/148) and [virtualpackage](https://github.com/chainguard-dev/tw/pull/147) into the same "package-type-checks" package and expose its use under different pipelines I'm ok with that too.

One benefit of putting the body of the script into a package is that it will get shellcheck run on it.